flake.nix

```diff
@@ -357,10 +372,9 @@
         tag = version;
         contents = [
           nickel
-          pkgs.bashInteractive
         ];
         config = {
-          Cmd = "bash";
+          Cmd = "${nickel}/bin/nickel";
```

Comment:
Should we use `Entrypoint` instead, for this kind of run-and-return CLI like Nickel?

Reply:
It looks like `Entrypoint` is indeed the right choice, cf. https://docs.docker.com/engine/reference/builder/#understand-how-cmd-and-entrypoint-interact.